Makefile

```
@@ -42,7 +42,12 @@ load-images:  ## Load images into the local kubernetes kind cluster.
 	./scripts/load-images.sh
 
 .PHONY: helm-install
-helm-install: cert-manager load-images  ## This is a phony target that is used to install the Tobs Helm chart.
+helm-install: start-kind cert-manager load-images  ## This is a phony target that is used to install the Tobs Helm chart.
+	helm dep up chart/
+	helm upgrade --install --wait --timeout 15m test chart/
```

```suggestion
	helm install --wait --timeout 15m test chart/
```

WDYT about making this an explicit `install` since this PR is also adding `helm-upgrade` target?

---

sounds good! Thanks